autoload/floaterm/cmdline.vim

```diff
@@ -15,18 +15,17 @@ function! floaterm#cmdline#parse(arglist) abort
   if a:arglist != []
     let c = 0
     for arg in a:arglist
-      if arg =~ '^--.*$'
+      if arg =~ '^--\S.*=.*$'
         let opt = split(arg, '=')
-        if len(opt) == 2
-          let [key, value] = [opt[0][2:], opt[1]]
-          if index(['height', 'width', 'autoclose'], key) > -1
-            let value = eval(value)
-          endif
-          let opts[key] = value
-        else
-          let key = opt[0][2:]
-          let opts[key] = v:true
+        if len(opt) != 2
+          call floaterm#util#show_msg('No value given to option: ' . opt[0], 'warning')
+          continue
```

Comment:
IMO it's better to abort and echo an error message. What's your opinion?S

Reply:
Yeah, I thought about it again and it makes more sense. So I will shift the `show_msg` call to `floaterm#run`.

Reply:
Actually it feels a bit silly to loop through `opts` again in `floaterm#run`. This is why we could use a more powerful parser that works for all the options (`opts` + `jobopts`) :joy: and maybe we don't even need `floaterm#run`.

Anyway, if I just swap `warning` -> `error` and `continue` -> `return` then it will abort, but it also prints the vim error from `floaterm#run`.

Reply:
OK, that will work for now. Sorry for the slow response, I was checking if there is a way to hide the `floaterm#run` errors.